product_docs/docs/pgd/5/parallelapply.mdx

```diff
@@ -40,11 +55,15 @@ SELECT bdr.alter_subscription_enable ('bdr_bdrdb_bdrgroup_node2_node1');
 
 ### When to use Parallel Apply
 
-Parallel Apply is always on by default and for most operations, we recommend that it is left on.
+Parallel Apply is always on by default and, for most operations, we recommend that it's left on.
+
+From PGD 5.2, Parallel Apply works with CAMO. It is not compatible with Group Commit or eager replication and should be disabled if Group Commit or eager replication are in use. 
```

Comment:
we don't say which commits scopes/replication types paralell apply can be used for before this - i believe it's asynch, legacy synch (which may or may not include sync commit) and now group commit)

Reply:
@cwcaro - do you know what things PA worked with correctly in the 5.0/5.1 series?

Reply:
Not sure, but I do know that PA also worked with streaming of large transactions